### PR TITLE
docs(layers_in_area): 13737 - readme added

### PR DESCRIPTION
**Pull request by Akiyamka**

### CODE REVIEW

File: src/features/layers_in_area/readme.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+## Layer in area
+
+This feature manage all layers that intersecting area of interest (a.k.a focused geometry).
```
Comment (vkozio) [inline src/features/layers_in_area/readme.md:3]: This feature manages what layers are available in area of interest (a.k.a focused geometry).

^ Is it better explanation?

File: src/features/layers_in_area/readme.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+Every time when geometry changes - this feature:
```
Comment (vkozio) [inline src/features/layers_in_area/readme.md:4]: probably needs more clarification what affects Layers in area

**Review by vkozio — COMMENTED** [review]
see comments above

File: src/features/layers_in_area/readme.md (hunk 1 above)
Comment (IlyaIzr) [reply src/features/layers_in_area/readme.md:3]: Yeah i think 'intersecting' is very specific word where's this feature manages all related to the AOI layers

File: src/features/layers_in_area/readme.md (hunk 2 above)
Comment (IlyaIzr) [reply src/features/layers_in_area/readme.md:4]:
```suggestion
Every time when focused geometry changes - this feature:
```

File: src/features/layers_in_area/readme.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+- generates and sends requests to the api
+- saves responses to a special cache
+- creates new logical layers
+- deletes the irrelevant layers
```
Comment (IlyaIzr) [inline src/features/layers_in_area/readme.md]: maybe smth like this?
```suggestion
- creates new logical layers if needed
- deletes the irrelevant layers if any
```

File: src/features/layers_in_area/readme.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+### How it works
+
+`layersGlobalResource` - load all world wide available layers.
+`layersInAreaAndEventLayerResource` - load event related and focused geometry related layers
+
+They load only info about what layers available, but not layers itself.
+
+When info about available layers loaded - `areaLayersControlsAtom` creates logical layers for each,
+so after that user/other features can enable or disable those layers.
```
Comment (IlyaIzr) [inline src/features/layers_in_area/readme.md]:
```suggestion
so after that user or other features can enable or disable those layers.
```

File: src/features/layers_in_area/readme.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+`areaLayersDetailsResource` - load data for **enabled** layers from above resources,
+that required for show layers legend, draw them on map, and so on.
```
Comment (IlyaIzr) [inline src/features/layers_in_area/readme.md]:
```suggestion
that required to show layers legend, draw them on map, and so on.
```
Also, isn't it all we get from details? Legend and source to generate legend and show layer on the mao?

File: src/features/layers_in_area/readme.md (hunk 2 above)
Comment (Akiyamka) [reply src/features/layers_in_area/readme.md:4]: good point, i will use vkozio variant

File: src/features/layers_in_area/readme.md (hunk 1 above)
Comment (Akiyamka) [reply src/features/layers_in_area/readme.md:3]: good point, i will use vkozio variant

**Pull request merged**